deploy.go

```diff
@@ -33,6 +33,7 @@ func deploy() {
 	go io.Copy(os.Stderr, stderr)
 	go io.Copy(os.Stdout, stdout)
 
-	cmd.Wait()
-
+	if err := cmd.Wait(); err != nil {
+		log.Fatal(err)
```

Comment:
can't you just exit with os.exit(1) instead? It might not be much use printing the `err` object which is a Go-specific data structure.

Reply:
Done